src/pages/patientView/PatientViewPage.tsx

```diff
@@ -538,6 +544,15 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </MSKTab>
                     )}
 
+                    {(getBrowserWindow().localStorage.trialmatch === 'true') && (
```

Comment:
you could put the check for localStorage up in shouldShowTrialMatch.  i realize it's there for testing purposes.

Reply:
Moved to shouldShowTrialMatch.